app/utils.py

```diff
@@ -49,9 +49,17 @@ def get_now_utc():
     def single_client_ip(request):
         if request['client_ip']:
             client_ip = request['client_ip']
-            ip_validation_pattern = re.compile(r'^[0-9.,\s]*$')
+            ip_validation_pattern = re.compile(r'^[0-9a-zA-Z.:,\s]*$')
```

Comment:
The '*' matches 0 or more repetitions of the character set, would "+" matching 1 or more repetitions be better?